apps/api/src/app/subscribers/e2e/update-preference.e2e.ts

```diff
@@ -31,7 +31,7 @@ describe('Update Subscribers preferences - /subscribers/:subscriberId/preference
 
     try {
       const response = await updatePreference(updateDataEmailFalse as any, session, template._id);
-      expect(response).to.not.be;
```

Comment:
we were missing the 'ok' in order test in order to trigger failure if the test is falsely passed.